src/renderer/helpers/api/local.js

```diff
@@ -551,6 +551,18 @@ function parseListItem(item) {
         descriptionShort: channel.description_snippet.text
       }
     }
+    case 'HashtagTile': {
+      /** @type {import('youtubei.js').YTNodes.HashtagTile} */
+      const hashtag = item
+
+      return {
+        type: 'hashtag',
+        title: hashtag.hashtag,
+        videoCount: parseLocalSubscriberCount(hashtag.hashtag_video_count.text),
+        channelCount: parseLocalSubscriberCount(hashtag.hashtag_channel_count.text),
```

Comment:
You could use `hashtag.hashtag_video_count.isEmpty()`, instead of adding an extra check in parseLocalSubscriberCount, if you want to.